frontend/TvCv19.Frontend/ClientApp/projects/patient/src/lib/patient-menu/patient-menu.component.html

```diff
@@ -0,0 +1 @@
+<p>patient-menu works!</p>
```

Comment:
can you implement this view

Reply:
I believe Jermey will be working on this. I will remove it from my PR.